docs/streams/upgrade-guide.html

```diff
@@ -173,6 +171,22 @@ <h3><a id="streams_api_changes_400" href="#streams_api_changes_400">Streams API
       See <a href="https://cwiki.apache.org/confluence/x/TZCMEw">KIP-1112</a> for more details.
     </p>
 
+    <h3><a id="streams_api_changes_400" href="#streams_api_changes_390">Streams API changes in 4.0.0</a></h3>
+
+    <p>
+        * Upgraded RocksDB dependency to version 9.7.3 (from 7.9.2). This upgrade incorporates various improvements and optimizations within RocksDB. However, it also introduces some API changes.
```

Comment:
Why are there `*` at the beginning of each line? This is regular HTML inside a <p> block, so having `*` seem not to be right?

If you want to use bullet points, you need to use the corresponding HTML markup.